cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -3551,39 +3815,43 @@ toShelleyAuxiliaryData m =
     Shelley.Metadata
       (toShelleyMetadata m)
 
-
 -- | In the Allegra and Mary eras the auxiliary data consists of the tx metadata
--- and the axiliary scripts.
---
-toAllegraAuxiliaryData :: forall era ledgerera.
-                          ShelleyLedgerEra era ~ ledgerera
-                       => Ledger.AuxiliaryData ledgerera ~ Allegra.AuxiliaryData ledgerera
-                       => Ledger.AnnotatedData (Ledger.Script ledgerera)
-                       => Map Word64 TxMetadataValue
-                       -> [ScriptInEra era]
-                       -> Ledger.AuxiliaryData ledgerera
-toAllegraAuxiliaryData m ss =
-    Allegra.AuxiliaryData
-      (toShelleyMetadata m)
-      (Seq.fromList (map toShelleyScript ss))
-
-
--- | In the Alonzo and later eras the auxiliary data consists of the tx metadata
+-- and the axiliary scripts. In the Alonzo and later eras the auxiliary data consists of the tx metadata
+-- and the axiliary scripts, and the axiliary script data. In the Alonzo and later eras the auxiliary data consists of the tx metadata
```

Comment:
Why are you repeating yourself here? (In the Alonzo and later eras...)

Reply:
Copy paste error